Author: Michael R. Crusoe <crusoe@debian.org>
Description: Remove the "static" feature to match liblzma-sys
Forwaded: not-needed

We only use the system liblzma, never static build
--- liblzma.orig/Cargo.toml
+++ liblzma/Cargo.toml
@@ -84,4 +84,3 @@
     "liblzma-sys/parallel",
     "num_cpus",
 ]
-static = ["liblzma-sys/static"]
